New serverless pattern - apigw (REST) - SQS - Lambda - IOT (#1796)
aws-samples/serverless-patterns: apigw-sqs-lambda-iot/README.md

# Amazon API Gateway (REST API) to Amazon SQS to AWS Lambda to AWS IOT 

> This pattern explains how to deploy a SAM application with  Amazon API Gateway (REST API), Amazon SQS, AWS Lambda and AWS IOT

Reviewer: ```suggestion
This pattern explains how to deploy a serverless application using SAM. The pattern connects Amazon API Gateway (REST API), Amazon SQS, AWS Lambda, and AWS IOT
```

> This pattern is useful to accept and respond to requests quickly but offloading the processing as asynchronous process. Once the request is placed in SQS, API gateway responds back to the caller immediately without waiting for those messages to be processed.

Reviewer: ```suggestion
This pattern accepts and respond to requests quickly by offloading the processing as asynchronous process. Once the request is placed in the SQS queue, API Gateway responds back to the caller immediately without waiting for those messages to be processed.
```

> When an HTTP POST request is made to the Amazon API Gateway endpoint, request payload is sent to Amazon Simple Queue Service. AWS Lambda function consumes event from the Queue and post the event/payload as MQTT message to AWS IoT Topic. 

Reviewer: ```suggestion
When an HTTP POST request is made to the Amazon API Gateway endpoint, request payload is sent to SQS queue. AWS Lambda function consumes event from the Queue and post the event/payload as MQTT message to AWS IoT Topic. 
```
